draft-ietf-rtcweb-jsep.xml

```diff
@@ -877,6 +877,63 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                 </section>
             </section>
+
+            <section  title="Simulcast" anchor="sec.simulcast">
+
+                <t>JSEP supports simulcast of a MediaStreamTrack, where
+                multiple encodings of the source media can be transmitted within
+                the context of a single m= section. The current JSEP API is designed
+                to allow applications to send simulcasted media but only to receive
+                a single encoding. This allows for multi-user scenarios where
+                each sending client sends multiple encodings to a server,
+                which then, for each receiving client, chooses the appropriate
+                encoding to forward.</t>
+
+                <t>Applications request support for simulcast by configuring
+                multiple encodings on an RTPSender, which, upon generation of
+                an offer or answer, are indicated in SDP markings on the
+                corresponding m= section, as described below.
+                Receivers that understand simulcast and are willing to receive it
+                will also include SDP markings to indicate their support, and JSEP
+                endpoints will use these markings to determine whether simulcast
+                is permitted for a given RTPSender. If simulcast support is not
+                negotiated, the RTPSender will only be permitted to use a single
+                encoding.</t>
```

Comment:
"Permitted" seems ambiguous here. Is the result that:

- You can add as many encodings as you want, but it will only use one.
- You won't even be able to add more encodings (say after the first O/A exchange)